packages/web-react/src/components/Skeleton/SkeletonText.tsx

```diff
@@ -0,0 +1,51 @@
+'use client';
+
+import classNames from 'classnames';
+import React, { ElementType } from 'react';
+import { SizesExtended } from '../../constants';
+import { useStyleProps } from '../../hooks';
+import { SpiritSkeletonProps } from '../../types';
+import SkeletonItem from './SkeletonItem';
+import { useSkeletonStyleProps } from './useSkeletonStyleProps';
+
+const defaultProps: Partial<SpiritSkeletonProps> = {
+  size: SizesExtended.MEDIUM,
+  lines: 1,
+  elementType: 'div',
+  ariaLive: 'polite',
+  ariaBusy: true,
+};
+const SkeletonText = <T extends ElementType = 'div', E = void>(
+  props: SpiritSkeletonProps<T, E>,
+): React.ReactElement => {
+  const propsWithDefaults = { ...defaultProps, ...props };
+  const {
+    elementType: ElementTag = defaultProps.elementType as ElementType,
+    size,
+    lines,
+    ariaLive,
+    ariaBusy,
+    isHeading,
+    ...restProps
+  } = propsWithDefaults;
+  const { classProps, props: modifiedProps } = useSkeletonStyleProps({ size, ...restProps });
```

Comment:
```suggestion
  const { classProps, props: modifiedProps } = useSkeletonStyleProps(restProps);
```
I think its not necessary to destructuring props here